tests/integration/210.command.deploy.test.cjs

```diff
@@ -437,7 +437,7 @@ if (process.env.NETLIFY_TEST_DISABLE_LIVE !== 'true') {
     })
   })
 
-  test.serial('should deploy functions from internal functions directory', async (t) => {
+  test.only('should deploy functions from internal functions directory', async (t) => {
```

Comment:
do you need to remove the `test.only` ?

Reply:
what a catch from thorley

Reply:
I commit `test.only` at least once a day